server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyContestResult.java

```diff
@@ -527,14 +525,8 @@ public void updateResults() {
       }
     }
     while (vote_total_iterator.hasNext()) {
-      // all the other choices that have the same number of votes as the last
-      // winner count as winners
-      entry = vote_total_iterator.next();
-      if (entry.getKey() == votes) {
-        my_winners.addAll(entry.getValue());
-      } else {
-        my_losers.addAll(entry.getValue());
-      }
+      // all the other choices count as losers
```

Comment:
Checking my understanding: the old code was wrong because, if we get into this loop, we have previously broken the previous `while` loop because `my_winners.size() >= my_winners_allowed` (if we broke out of the previous `while` loop because we exhausted the iterator, we'll never enter this one). That means that found as many winners as we are allowed to find. Thus, there's no way (no matter how many votes were earned) anything else could be a winner.

That would make sense, except what if there are multiple ties for various places? Then what we call "winners" are just determined by the iteration order, right? For concreteness, suppose we have a contest with 2 winners allowed. The votes are:
- `A` has 10 votes
- `B` has 5 votes
- `C` has 5 votes
- `D` has 2 votes

Then we'd call `A` and `B` winners, stop iterating, and call `C` and `D` losers, even though `B` and `C` are tied. Does some other precondition prevent us from reaching this scenario?

Reply:
We _would_, in that case, call `C` and `D` losers, but we'd know there was a tie, the minimum margin would be 0, and the contest would be unauditable. It is, essentially irrelevant which members of a tied group are called "winners" and which are called "losers", because any time a tie exists, the contest is not auditable.

Reply:
(unless the tie is between losers, of course)